driver/cassandra/cassandra.go

```diff
@@ -83,16 +58,16 @@ func (driver *Driver) Close() error {
 }
 
 func (driver *Driver) ensureVersionTableExists() error {
-	err := driver.session.Query("CREATE TABLE IF NOT EXISTS " + tableName + " (version counter, versionRow bigint primary key);").Exec()
-	if err != nil {
-		return err
-	}
 
-	_, err = driver.Version()
-	if err != nil {
-		driver.session.Query(up.String(), versionRow).Exec()
+	if err := driver.session.Query("CREATE TABLE IF NOT EXISTS " + tableName +
+		" (driver_version int," +
+		"version bigint," +
+		"file_name text," +
```

Comment:
We need a `where` clause when we use `order by` with cassandra.
As I don't know when the migration are applied, I use this version trick to make it work.
In fact, if you look at the previous version of this driver, you will see there is a `const version` too.